### DISCUSSION TITLE
Node setup

### DISCUSSION THREAD

**Discussion by sankararaokgmail** [filecoin-project/lotus#6174] [General Q&A] [answered]
I would like to set up lotus mainnet. can someone let me know the size of the node and the size that it may grow per day?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
the full node is around 5.1TB - it grows aroudn 35Gb /day right now

the snapshot node (from an imported snapshot) is way way smaller and the size depends on how ofter you reimport new snapshots

see https://docs.filecoin.io/get-started/lotus/chain/
